src/www/ui/api/Controllers/FileInfoController.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/*
+ SPDX-FileCopyrightText: © 2018 Siemens AG
+ Author:  Author: Soham Banerjee <sohambanerjee4abc@hotmail.com>
+ SPDX-FileCopyrightText: © 2023 Soham Banerjee <sohambanerjee4abc@hotmail.com>
+ SPDX-License-Identifier: GPL-2.0-only
+*/
+
+/**
+ * @file
+ * @brief Controller for copyright queries
+ */
+
+namespace Fossology\UI\Api\Controllers;
+
+use Fossology\Lib\Auth\Auth;
+use Fossology\Lib\Dao\UploadDao;
+use Fossology\Lib\Db\DbManager;
+use Fossology\UI\Api\Helper\ResponseHelper;
+use Fossology\UI\Api\Models\Info;
+use Fossology\UI\Api\Models\InfoType;
+use Psr\Http\Message\ServerRequestInterface;
+
+
+class FileInfoController extends RestController
+{
+  /**
+   * @var ContainerInterface $container
+   * Slim container
+   */
+  protected $container;
+
+  /**
+   * @var ClearingDao
+   */
+  private $clearingDao;
+
+  /**
+   * @var LicenseDao $licenseDao
+   * License Dao object
+   */
+  private $licenseDao;
+
+  /**
+   * @var UploadDao $uploadDao
+   * Upload Dao object
+   */
+  private $uploadDao;
+
+  /**
+   * @var viewInfo $viewInfo
+   * View Info object
+   */
+  private $viewInfo;
+
+  public function __construct($container)
+  {
+    parent::__construct($container);
+    $this->container = $container;
+    $this->clearingDao = $this->container->get('dao.clearing');
+    $this->licenseDao = $this->container->get('dao.license');
+    $this->uploadDao = $container->get('dao.upload');
+    $this->restHelper = $container->get('helper.restHelper');
```

Comment:
Setting container is already done in parent class.
```suggestion
    $this->clearingDao = $this->container->get('dao.clearing');
    $this->licenseDao = $this->container->get('dao.license');
    $this->uploadDao = $this->container->get('dao.upload');
    $this->restHelper = $this->container->get('helper.restHelper');
```